content/en/docs/reference/command-line-tools-reference/feature-gates/retry-generate-name.md

```diff
@@ -9,6 +9,14 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.30"
+    toVersion: "1.31"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.31"
```

Comment:
Please double check. If the gate is beta from 1.31, then line 12 should be 1.30.
Similarly, if the gate is GA in v1.32, then line 16 should be 1.31.

Reply:
I see, the ranges are inclusive. Fixed. Thanks for catching.